edi_storage_oca/migrations/16.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+from odoo.tools.sql import create_column
+
+from odoo.addons.http_routing.models.ir_http import slugify
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not version:
+        return
+
+    required_modules = ("storage_backend", "edi_storage_oca")
+    if not env["ir.module.module"].search(
```

Comment:
I think you don't need this verificatiion. If you are in this script, you come from 15, so `storage_backend` is installed